core/modules/config/config.module

```diff
@@ -6,6 +6,11 @@
 
 /**
  * Implements hook_menu().
+ * 
+ * @see config_set()
+ * @see config_get()
+ * @see config_clear()
+ * @see config_diff()
```

Comment:
`config.inc` in core provides the main `config_set/get/clear()` functions. We don't do this for any other hook_menu() in Backdrop, e.g. `node_menu()` doesn't `@see` reference `node_load()` and `views_menu()` doesn't reference `views_get_view()`.

Reply:
done too